master/internal/api_shell.go

```diff
@@ -125,6 +153,17 @@ func (a *apiServer) LaunchShell(
 		return nil, api.APIErrToGRPC(errors.Wrapf(err, "failed to prepare launch params"))
 	}
 
+	workspaceID := model.DefaultWorkspaceID
```

Comment:
did you wanna use `isNTSCPermittedToLaunch` ?

Reply:
just added!